playground/backend/internal/preparers/python_preparers_test.go

```diff
@@ -101,3 +95,129 @@ func Test_addCodeToFile(t *testing.T) {
 		})
 	}
 }
+
+func Test_saveLogs(t *testing.T) {
+	file, _ := os.Open(correctPyFile)
+	tmp, _ := utils.CreateTempFile(correctPyFile)
+	defer tmp.Close()
+
+	type args struct {
+		from *os.File
+		to   *os.File
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			// Call saveLogs method to add logs code to tmp file.
+			name: "Save logs successfully",
```

Comment:
Let's also add a negative test case when we try to save logs to a non-existing file.

Reply:
Done

Reply:
Sorry, could you please point me to it? Can't find the test case